tests/THORChain/TWSwapTests.cpp

```diff
@@ -204,13 +204,13 @@ TEST(TWTHORChainSwap, SwapBnbBtc) {
     Binance::Proto::SigningInput txInput = outputProto.binance();
     
     // set few fields before signing
-    txInput.set_chain_id("Binance-Chain-Nile");
```

Comment:
huh, I didn't know we hardcode testnet chain id if not set, how does thorchain swap tx work if wrong chain id?

Reply:
The hardcoded value was only in the buildTransactionInput(), which is the external signature use case, not used commonly, and only in the case it was not set.

Reply:
For the THORSwap use case, after the SigningInput is built, a few fields have to be set, incl. chainId, this was so.